amplifiers/elecraft/kpa.c

```diff
@@ -0,0 +1,593 @@
+/*
+ *  Hamlib Elecraft amplifier backend - low level communication routines
+ *  Copyright (c) 2019 by Michael Black W9MDB
+ *
+ *
+ *   This library is free software; you can redistribute it and/or
+ *   modify it under the terms of the GNU Lesser General Public
+ *   License as published by the Free Software Foundation; either
+ *   version 2.1 of the License, or (at your option) any later version.
+ *
+ *   This library is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *   Lesser General Public License for more details.
+ *
+ *   You should have received a copy of the GNU Lesser General Public
+ *   License along with this library; if not, write to the Free Software
+ *   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include "kpa.h"
+
+struct fault_list
+{
+  int code;
```

Comment:
Formatting issues. Much of the new code uses 2 spaces instead of 4. Also, the braces with if/else/while statements do not follow the documented formatting rules. We should automate the formatting process, so we could do it with a simple script for every PR before merged.